### PR TITLE
Bug 1393337 Added procedure to the upgrading OS topic

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1393337

Added the desired procedure to safely upgrade a host, after conversations with BZ reporter and devs.

I'll ask for an ack in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
Passed review.

@ahardin-rh @adellape @bmcelvee peer review please?

File: install_config/upgrading/os_upgrades.adoc [hunk 1]
```diff
@@ -7,26 +7,54 @@
...
-== Solutions
-
-Reboot the node to ensure that all of the software is running the new
-versions, especially if a new kernel is installed. A reboot will also ensure
-that the `docker` and {product-title} processes have been restarted, which will
-force them to check that all of the rules in other services are correct.
-
-However, if you do not wish to do a reboot of the node, then it is possible to
-just restart the services that are affected, or to preserve the `iptables`
-state. Both processes are described in the
```
Comment (ahardin-rh) [inline install_config/upgrading/os_upgrades.adoc]: need a section ID here

**Comment by ahardin-rh** [comment]
Just one comment from me. Otherwise, looks good! :rocket:

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/upgrading/os_upgrades.adoc#install-config-upgrading-os-upgrades[Upgrading a Cluster -> Operating System Updates and Upgrades]
|Added the procedure for upgrading the operating system.
%

**Comment by bfallonf** [comment]
Chatted that this section's ok without an anchor.

I'll merge.

**Comment by adellape** [comment]
> Chatted that this section's ok without an anchor.
> 
> I'll merge.

@bfallonf @vikram-redhat What was the reasoning in the chat? I feel like we should go ahead and put unique anchors for any section even if we're not linking to it necessarily yet, in case it's linked to later and for consistency (sticking to our guidelines). WDYT?

**Comment by bfallonf** [comment]
@adellape Because it's the only section in this topic, I didn't think it needed it, and I talked around to people and they sorta felt the same.

I'll send you an email.

**Pull request merged**